climpred/__init__.py

```diff
@@ -1,5 +1,5 @@
 # flake8: noqa
-from pkg_resources import DistributionNotFound, get_distribution
+from importlib_metadata import distribution
```

Comment:
I think you can use `importlib.metadata` from the standard library when requiring python 3.8+

```python
from importlib.metadata import version as _get_version
__version__ = _get_version("climpred")
```